documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -823,10 +823,11 @@ consistency is sacrificed in the presence of partition(s). Below is a list of th
 
 | ALLOW_READS
 | Allows reads for a given key if it exists in this partition, but only allows writes if this partition contains all owners of a segment.
-| Availability
+This is still a consistent approach because some entries are readable if available in this partition, but from a client application perspective it is not deterministic.
+| Consistency
 
 | ALLOW_READ_WRITES
-| Allow entries on each partition to diverge, with conflicts resolved during merge.
+| Allow entries on each partition to diverge, with the possibility of conflicts resolved during merge.
```

Comment:
"Allow entries on each partition to diverge, with conflict resolution attempted upon the partitions merging."